build.config.json

```diff
@@ -1,6 +1,7 @@
 {
     "appId": "com.electron.hugin",
-    "productName": "Hugin Messenger",
+    "productName": "HuginMessenger",
```

Comment:
I dont think this should change! I suspect it might break and/or collide with old Hugin folders in %AppData%